packages/twenty-server/src/engine/api/graphql/graphql-query-runner/helpers/process-aggregate.helper.ts

```diff
@@ -28,7 +29,15 @@ export class ProcessAggregateHelper {
         aggregatedField.fromField,
         aggregatedField.fromSubField,
       );
-      const operation = aggregatedField.aggregationOperation;
+      const operation = aggregatedField.aggregateOperation;
```

Comment:
Can we keep aggregateOperation throughout the code ? Operation alone loses its meaning.